thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detection/InputDataSpec.java

```diff
@@ -21,20 +21,23 @@
 import java.util.Collections;
 
 
-public class StaticDetectionPipelineModel {
+/**
+ * The data spec to describe all the input data for a detection stage perform the detection.
+ */
+public class InputDataSpec {
   final Collection<MetricSlice> timeseriesSlices;
```

Comment:
can you add java doc for each input? what is aggregateSlice

Reply:
sure. I'll add the java doc.